### PR TITLE
Fix two crashes

### PR DESCRIPTION
**Pull request by gradientvera**

### CODE REVIEW

File: NFluidsynth/Native/LibFluidsynth.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ static LibFluidsynth()
                 {
                     if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
                     {
-                        return NativeLibrary.Load("/home/pj/Projects/fluidsynth/build/src/libfluidsynth.so");
+                        return NativeLibrary.Load("libfluidsynth.so");
```
Comment (PJB3005) [inline NFluidsynth/Native/LibFluidsynth.cs:22]: Ah fuck.

Comment (gradientvera) [reply NFluidsynth/Native/LibFluidsynth.cs:22]: Yeah, I had to change it or it'd crash oof

File: NFluidsynth/Native/LibFluidsynth.Misc.cs [hunk 2]
```diff
@@ -6,14 +6,14 @@ namespace NFluidsynth.Native
     internal static partial class LibFluidsynth
     {
         [DllImport(LibraryName)]
-        [return: MarshalAs(UnmanagedType.I4)]
+        [return: MarshalAs(UnmanagedType.I1)]
```
Comment (PJB3005) [inline NFluidsynth/Native/LibFluidsynth.Misc.cs]: Hrm. This *shouldn't* be correct as the Fluidsynth source code definitely says it's of type `int`.

I would prefer you made it `int` instead and made usages check instead.

Comment (gradientvera) [reply NFluidsynth/Native/LibFluidsynth.Misc.cs]: Yeah, but for some reason it crashed on my computer, complaining about it being I4...
I've changed it to return int now

**Pull request merged**